src/triage.rs

```diff
@@ -17,6 +18,34 @@ pub fn index() -> Result<Response<Body>, hyper::Error> {
         .unwrap())
 }
 
+pub async fn asset(file: &str) -> Result<Response<Body>, hyper::Error> {
```

Comment:
The backoffice started as an HTML page with a separate JavaScript and CSS file, so I thought to add a static asset endpoint. Then I moved everything to the server, the JavaScript was basically removed. scripts and stylesheets are now embedded in the HTML file (~4kb with gzip compression).

This asset path is now only used to serve 2 SVG. Unsure if we want to keep it after all :)